common/constants.py

```diff
@@ -14,3 +14,6 @@
     USE_ALL_LOGS = True
 else:
     USE_ALL_LOGS = False
+
+# maximum number of chars to print when a matching text is detected.
+MAX_MATCH_CHARS = 200
```

Comment:
I don't think this should be super global. Maybe better to make it Juju specific for now.